core/config/cb_config.go

```diff
@@ -176,3 +176,9 @@ func getFallbackPolicySpec(command string) *model.FallbackPolicySpec {
 	}
 	return GetHystrixConfig().FallbackPolicyProperties.Provider
 }
+
+// GetForceFallback get force fallback
+func GetForceFallback(command, t string) bool {
```

Comment:
since default value is false， then don‘t use archaius to get value but use struct pointer

Reply:
Done